dev-tools/ecs-migration.yml

```diff
@@ -142,31 +142,31 @@
 
 ## Apache
 
-- from: apache2.access.user_name
+- from: apache.access.user_name
```

Comment:
So this becomes now interesting. This is for compatibility with 6.x. In 6.x it will stay apache2. Now comes the tricky part: We will have a mix of apache2 and apache fields. What should we do?

Reply:
Not only the fields, but also configurations referring to both names should work during at least some versions (6.7?).
Shall we duplicate the module during some versions?

Reply:
@ruflin easier, I have reverted the changes in the ECS migration, and I have added aliases for the rest of fields, so we use the same strategy than for ECS.

Reply:
For config migration I am still not sure how to do it.

Reply:
@ruflin I am trying the approach of adding an option to `module.yml` to indicate that a module has been moved, this keeps old configurations working, see 9af3f75a9f68dd2f3a4b166ed2d8bb9ca1e588ce.
If you are ok with this approach I'll open an specific PR just for that.

Reply:
PR for moved modules #9432